sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-energy-demand/automl-forecasting-task-energy-demand-advanced-mlflow.ipynb

```diff
@@ -136,32 +135,76 @@
     "Next, we will upload the data by creating an MLTable to be used for training.\n",
     "\n",
     "Please make use of the MLTable files present within the data folder at the same location (in the repo) as this notebook."
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "# Training MLTable defined locally, with local data to be uploaded\n",
     "my_training_data_input = Input(\n",
     "    type=AssetTypes.MLTABLE, path=\"./data/training-mltable-folder\"\n",
     ")"
-   ]
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "my_training_data_input.__dict__"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
-    "Next, we upload the directory with the test set data which will be used in the batch end point inference."
-   ]
+    "To create data input from TabularDataset created using V1 sdk, specify the `type` as `AssetTypes.MLTABLE`, `mode` as `InputOutputModes.DIRECT` and `path` in the following format `azureml:<tabulardataset_name>:<version>`."
```

Comment:
Hi Nikolay, this format azureml:<tabulardataset_name>:<version> is also in the notebooks of other verticals. Would it be better if I reframe it as : "To create data input from TabularDataset created using V1 sdk, set the type to AssetTypes.MLTABLE, mode to InputOutputModes.DIRECT and path to the following format azureml:<tabulardataset_name> or azureml:<tabulardataset_name>:<version> (in case we want to use specific version). ".
Please let me know.

Reply:
Yes, we can use this wording.